packages/checkPrescriptionStatusUpdates/tests/testHander.test.ts

```diff
@@ -92,3 +183,20 @@ describe("test handler", () => {
     })
   })
 })
+
+function generate_reply(count, final = true) {
```

Comment:
Mixing snake and camel case here.

Reply:
cf21c20c5378acf75c3ac684ee30ea95b3e4ad76